bin/zopen-audit

```diff
@@ -119,15 +119,21 @@ while IFS= read -r repo; do
     continue
   fi
 
-  commit_sha=$(jq -er '.product.community_commitsha' "${pkghome}/metadata.json")
+  release=$(jq -er '.product.release' "${pkghome}/metadata.json")
   if [ $? -gt 0 ]; then
-    printVerbose "No community_commitsha in $repo"
+    printVerbose "No release timestamp in $repo"
     continue
   fi
 
-  # Fetch CVEs using the commit_sha from the vulnerabilities JSON
-  cves=$(jq -cer 'if .["'$repo'"] then .["'$repo'"][] | select(.commit_sha == "'$commit_sha'") | .CVEs? // [] | [.severity, .id, .details] | @tsv else empty end' $JSON_CVE_CACHE)
-  if [ $? -gt 0 ] || [ -z "$cves" ]; then
+  # Fetch CVEs using the release timestamp from the vulnerabilities JSON
+  cves=$(jq -cr '.["'$repo'"] // {} | .["'$release'"] // {} | .CVEs? // [] | .[] | [.severity, .id, .details] | @tsv' $JSON_CVE_CACHE)
```

Comment:
Prior to adding `if .["'$repo'"] then .`, I hit some `null` related errors with jq. Does `zopen audit` work fine without it on z/OS?

Reply:
It works fine without it in my testing -- the `//` operator in jq allows you to choose an alternate value if the key doesn't exist, so if `.["'$repo'"]` is `null` for example then it will output an empty object instead. This way avoids having several nested `if then`s, which got a little bit messy.

Reply:
Great!